distmv/normal.go

```diff
@@ -246,6 +246,29 @@ func (n *Normal) Prob(x []float64) float64 {
 	return math.Exp(n.LogProb(x))
 }
 
+// Quantile returns the multi-dimensional inverse cumulative distribution function.
+// len(x) must equal len(p), and if x is non-nil, len(x) must also equal len(p).
+// If x is nil, a new slice will be allocated and returned, otherwise the quantile
+// will be stored in-place into x. All of the values of p must be between 0 and 1,
+// or Quantile will panic.
+func (n *Normal) Quantile(x, p []float64) []float64 {
+	dim := n.Dim()
+	if len(p) != dim {
+		panic(badSizeMismatch)
```

Comment:
What is `badSizeMismatch` supposed to be used for? I think we should bring these constants together - having them spread around in separate files the way they are leads to this kind of problem.